src/DIRAC/DataManagementSystem/scripts/dirac_dms_replicate_and_register_request.py

```diff
@@ -22,15 +22,19 @@ def getLFNList(arg):
 def main():
     catalog = None
     Script.registerSwitch("C:", "Catalog=", "Catalog to use")
+    Script.registerSwitch("N:", "ChunkSize=", "Number of files per request")
```

Comment:
```suggestion
    Script.registerSwitch("N:", "ChnkSze=", "Number of files per request")
```

To not confuse it with line 36 and 37!